sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -217,6 +217,18 @@ object ScalaJSPlugin extends AutoPlugin {
         "`scalaJSUseMainModuleInitializer` is true",
         CTask)
 
+    val scalaJSLinkerConfig = SettingKey[StandardLinker.Config](
+        "scalaJSLinkerConfig",
+        "Configuration of the Scala.js linker",
+        BPlusSetting)
+
+    val scalaJSLinkerBatchMode = SettingKey[Boolean](
+        "scalaJSLinkerBatchMode",
+        "Whether to run the Scala.js linker in batch mode, where nothing is " +
+        "cached nor incremental. This reduces the memory footprint, which " +
+        "can be particularly useful in a CI environment.",
+        BMinusSetting)
```

Comment:
It is `ClearableLinker` which "disables caching" in batch mode, because it throws away the linker after every `link`. That's a nice thing about `ClearableLinker`, the the implementation of `Linker` it wraps doesn't have to deal with batch mode or not.